src/Mono.Android.Export/Properties/AssemblyInfo.cs

```diff
@@ -9,6 +10,7 @@
 [assembly: AssemblyTrademark ("Xamarin")]
 [assembly: AssemblyCulture ("")]
 [assembly: AssemblyMetadata ("IsTrimmable", "True")]
+[assembly: SupportedOSPlatform ("Android21.0")]
```

Comment:
I think it might be better to put something like this in the `.csproj`?
```xml
<SupportedOSPlatformVersion>$(AndroidMinimumDotNetApiLevel)</SupportedOSPlatformVersion>
```
I'm also not sure, but the MSBuild property might enable more build analyzers that if you just have it in `.cs` source files.

Reply:
Oh maybe you have to do it this way... The project is `net6.0` and not `net6.0-android`.